File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 5 - Python/q8.py
#Escreva um algoritmo para ler 10 números. Todos os números lidos com valor inferior a 40 devem
#ser somados. Escreva o valor final da soma efetuada.

soma=0
for i in range(5):
    x=int(input("Insira um número: "))
    if x<40:
        soma=soma+x
print("Soma: ", soma)

File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 5 - C/q8.c
//Escreva um algoritmo para ler 10 números. Todos os números lidos com valor inferior a 40 devem
//ser somados. Escreva o valor final da soma efetuada.
#include<stdio.h>
int main(){
	int x, soma, i;
	soma=0;
	for(i=1; i<=5; i++){
		printf("Insira um numero: ");
		scanf("%d", &x);
		if(x<40){
			soma=soma+x;	
		} 
	}
	printf("Soma: %d", soma);	
}
